conflictlog/pool.go

```diff
@@ -31,7 +32,7 @@ type ConnPool interface {
 	// Get returns an object from the pool. If there is none then it creates
 	// one by calling newConnFn() and returns it. It is guaranteed that either
 	// an error or a non-nil connection object will be returned
-	Get(bucketName string) (conn io.Closer, err error)
+	Get(bucketName string, timeout time.Duration) (conn io.Closer, err error)
```

Comment:
In comment above, describe what the `timeout` is for
i.e. `ErrConnPoolGetTimeout` is returned